appservice/src/intTest/groovy/nu/yona/server/AbstractAppServiceIntegrationTest.groovy

```diff
@@ -67,4 +67,18 @@ abstract class AbstractAppServiceIntegrationTest extends Specification
 		int num = sequenceNumber++
 		return "$baseTimestamp$num"
 	}
-}
+
+	def assertDateTimeInLimits(dateTimeString, int rangeSeconds = 10, Date comparisonDateTime = null)
```

Comment:
I don't think that's a good idea:
* The ``equals`` part of the name conflicts with the range. If the range is non-zero, then it's a range or limits assertion.
* Making the default zero is a call for trouble for true client/server testing. Currently, the tests run against local host, but we must also be able to run them against a remote host. In that case, clock synchronization will be in place, but not up to the microsecond. That's why I made the range default 10 seconds.